### PR TITLE
data kingDOM2227

### PR DESCRIPTION
**Pull request by kingDOM2227**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4159"></a>

### CODE REVIEW

File: data/kingDOM2227.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source DevRel | Python Developer | Data Scientist  | **GitHub** Star",
+  "avatar": "https://github.com/kingDOM2227.png",
+  "tags": [
+    "Open Source",
+    "Python",
+    "R",
+    "Java",
+    "Javascript",
```
Comment (swagataroy30) [inline data/kingDOM2227.json:12]:
```suggestion
    "Javascript"
```

**Review by swagataroy30 — COMMENTED** [review]
Can you please remove the comma from line 12. Thanks you!

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

In the recent update, we don't need to add `avatar:` anymore, it will auto-pick your GitHub profile picture. So, please remove that. If you need any help feel free to ask.

Updated docs: https://linkfree.eddiehub.io/docs/quickstart

File: data/kingDOM2227.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Abiola John Oluwaseyi",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source DevRel | Python Developer | Data Scientist  | **GitHub** Star",
+  "avatar": "https://github.com/kingDOM2227.png",
```
Comment (Pradumnasaraf) [inline data/kingDOM2227.json:6]:
```suggestion
```

File: data/kingDOM2227.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Abiola John Oluwaseyi",
+  "type": "personal",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/kingDOM2227.json:4]: Following an improvement, this is no longer needed
```suggestion
```

**Pull request merged**